src/AdvancedMH.jl

```diff
@@ -9,7 +9,7 @@ import Random
 
 # Exports
 export MetropolisHastings, DensityModel, RWMH, StaticMH, StaticProposal, 
-    RandomWalkProposal, Ensemble, StretchProposal
+    RandomWalkProposal, Ensemble, StretchProposal, MALAMH, MALA
```

Comment:
```suggestion
    RandomWalkProposal, Ensemble, StretchProposal, MALA
```

See my comment below about removing `MALAMH` in favor of just `MALA`.